apiserver/plane/app/serializers/project.py

```diff
@@ -24,54 +24,50 @@ class Meta:
         fields = "__all__"
         read_only_fields = ["workspace", "deleted_at"]
 
-    def create(self, validated_data):
-        identifier = validated_data.get("identifier", "").strip().upper()
-        if identifier == "":
-            raise serializers.ValidationError(detail="Project Identifier is required")
-
-        if ProjectIdentifier.objects.filter(
-            name=identifier, workspace_id=self.context["workspace_id"]
-        ).exists():
-            raise serializers.ValidationError(detail="Project Identifier is taken")
-        project = Project.objects.create(
-            **validated_data, workspace_id=self.context["workspace_id"]
+    def validate_name(self, name):
+        project_id = self.context.get("project_id")
```

Comment:
@sangeethailango You don't have to get this from context. `instance` will be available at `self.instance`, you can use that to exclude the current project from the check